styles.css

```diff
@@ -0,0 +1,140 @@
+@import url("https://rsms.me/inter/inter.css");
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Inter", sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+}
+
+/* navBar */
+
+ul {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  font-weight: 500;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+li {
+  padding: 1%;
+  color: #172b4d;
+}
+
+a {
+  text-decoration: none;
+  color: #172b4d;
+}
+
+/* the whole content */
+
+.content {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  background-image: url(./utils/Header-llustration-desktop@2x.png);
+  height: 100vh;
+  width: 100%;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 1970px 1100px;
+  background-attachment: fixed;
+  background-position-y: top;
+}
+
+.welcome-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: 30px;
+  padding-left: 10%;
+  max-width: 90%;
+  padding-right: 10%;
+  margin-top: 15%;
+}
+
+.welcome-section > * {
+  margin-top: 15px;
+}
+
+.title {
+  font-size: 60px;
+  color: #172b4d;
+}
+
+.subTitle {
+  color: #36b37e;
+}
+
+.bio {
+  color: #172b4d;
+  font-weight: 200;
+  max-width: 70%;
+  text-align: center;
+}
+
+.socials {
+  display: flex;
+  margin-top: 50px;
+}
+
+.fa {
+  justify-content: space-between;
+  margin-right: 15px;
+  margin-left: 15px;
+}
```

Comment:
Kindly adjust the size of the hamburger button to match the design. Check the above comment for more on why you should stick to the given design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185383912-4a6d377b-e4da-4175-a92e-e196ad953623.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/185383972-f47c9033-fd2c-439c-a8d8-da88225e4aec.png)